pkg/inst/worker/worker.R

```diff
@@ -42,11 +38,11 @@ for (pkg in packageNames) {
 }
 
 # read function dependencies
-depsLen <- SparkR:::readInt(inputCon)
-if (depsLen > 0) {
-  execFunctionDeps <- SparkR:::readRawLen(inputCon, depsLen)
-  # load the dependencies into current environment
-  load(rawConnection(execFunctionDeps, open='rb'))
+funcLen <- SparkR:::readInt(inputCon)
+if (funcLen > 0) {
```

Comment:
I think this will always be greater than 0 ? If not we don't have any function to compute so we should at least have `computeFunc <- identity` or something like that in the else clause